### PR TITLE
Improve the provenance of the XML:DB API dependency

### PR DESCRIPTION
**Pull request by adamretter**
Jar Binary from the established source code provenance - github.com/exist-db/xmldb-org

### CODE REVIEW

File: src/org/exist/xmldb/LocalResourceSet.java [hunk 1]
```diff
@@ -87,6 +87,13 @@ public void addResource(final Resource resource) throws XMLDBException {
         resources.add(resource);
     }
 
+    @Override
```
Comment (dizzzz) [inline src/org/exist/xmldb/LocalResourceSet.java]: why is this needed?

Comment (adamretter) [reply src/org/exist/xmldb/LocalResourceSet.java]: The "official" API has this, it was really the only difference apart from `int` vs `long` for some sizes.
